tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -535,4 +535,44 @@ TEST_P(BackendCorrectnessTest, maxSplatTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// Symmetric Quantized Conv+Relu fusion testing for only OpenCL.
+TEST_P(BackendCorrectnessTest, SymmetricQuantizedConvReluFusionTest) {
+  CHECK_IF_ENABLED()
+  PseudoRNG PRNG;
+  Tensor inputs(ElemKind::Int8QTy, {1, 6, 6, 16}, 1, 0);
+  Tensor kernel(ElemKind::Int8QTy, {1, 3, 3, 16}, 1, 0);
+  Tensor bias(ElemKind::Int32QTy, {1}, 1, 0);
+  inputs.getHandle<int8_t>().randomize(0, 60, PRNG);
+  kernel.getHandle<int8_t>().randomize(-1, 1, PRNG);
+  bias.getHandle<int32_t>().randomize(0, 32768, PRNG);
+  std::array<dim_t, 4> S{{1, 6, 6, 1}};
+  llvm::ArrayRef<dim_t> shape(S);
+  Tensor out(ElemKind::Int8QTy, shape, 256, 0);
```

Comment:
nit: We should be able to just pass S into the Tensor constructor, no? Arrays should implicitly get converted to ArrayRefs.

Reply:
I changed it directly to convey the shape of the tensor to the constructor.